bookkeeper-server/src/main/java/org/apache/bookkeeper/proto/BookieProtocol.java

```diff
@@ -432,10 +433,7 @@ public String toString() {
                                  opCode, ledgerId, entryId, errorCode);
         }
 
-        void retain() {
-        }
-
-        void release() {
```

Comment:
what about keeping `boolean release()`
IIUC your problem is about implementing `ReferenceCounted`

Reply:
It has conflict with the release() interface in `ReferenceCounted`

Reply:
Yeah. I think we can keep `boolean release()`. Because `ReferenceCounted` also has the same method.

Reply:
👍